### DISCUSSION TITLE
Publishing test result failed

### DISCUSSION THREAD

**Discussion by Dr-mkelvo** [DevSkillsHQ/discusssions#78] [Help]
I have a problem with the script that should publish my code to Alva. The script runs npm install without checking for incompatible modules, which causes the code to fail when publishing to Alva results. Can you do anything about that? I've tried upgrading everything while downgrading the versions of the packages to fix the dispute, but it persists. My current approach involves using --legacy-peer-deps. 
NB I'm using the frontend-boilerplate-react-nextjs.
![image](https://github.com/DevSkillsHQ/discusssions/assets/118157498/db5c32b7-65fe-4d80-b7af-7d254b97b499)

**Comment by ehsan-alva** [comment]
To resolve the issue, you should adjust the packages within the /app-frontend folder. Please revert any modifications you've made to package.json outside of this scope. This action should help in addressing the problem.

**Reply by Dr-mkelvo** [reply]
Thank you @ehsan-alva, It worked.
